README.md

```diff
@@ -175,26 +171,22 @@ foreach($weak_references as $weak_ref_object_handle => $weak_reference) {
 
 ## Development and testing
 
-This extension shipped with Vagrant file which provides basic environment for development and testing purposes. 
-To start it, just type `vagrant up` and then `vagrant ssh` in php-weak directory.
+This extension ships with a Vagrant file which provides a basic environment for development and testing purposes. 
+To start it, just type `vagrant up` and then `vagrant ssh` from within the main directory.
 
 Services available out of the box are:
 
  - Apache2 - on [192.168.33.10:8080](http://192.168.33.102:8080)
  - nginx - on [192.168.33.10:80](http://192.168.33.102:80)
 
-For plumbing memory-related problems there are valgrind, to activate it, execute `export TEST_PHP_ARGS=-m` before running tests.
+For debugging memory-related problems, valgrind is already installed. To activate it, execute `export TEST_PHP_ARGS=-m` before running tests.
 
-To prevent asking test suite to send results to PHP QA team, `NO_INTERACTION=1` env variable is set. If run tests in your
-own environment, just execute `export NO_INTERACTION=1` to mute that reporting.
+The test suite may prompt to send results to the PHP QA team, which can be disabled by setting the `NO_INTERACTION=1` environment variable. If you run tests in your
+own environment, the shell command `export NO_INTERACTION=1` will disable it.
 
-There are `php7debugzts` directory inside this repo which contains include files for master php branch (which may be out
-of date sometimes) for type-hinting inside your IDE in case you don't have PHP 7 include files installed. They are
-provided only for typehinting and nothing more. While for quick fixes they are ok, for real development you have to point
-to real file from the same PHP version you are building extension for.
+There is a `php7debugzts` directory inside this repo which contains include files for the master php branch (which may be out of date sometimes) to support type-hinting inside your IDE in case you don't have PHP7 include files installed. They are provided only as a convenience, and any serious deveopment should use the real files from the exact PHP version you are developing against.
```

Comment:
I have feedback from php internals devs that it is not very wise idea to ship it and I agree with them. It also pollute diffs significantly, so I guess I will remove it and force-push repo (incl tags), it is wrong, but as long as no functionality touched, i guess, it is ok.